platforms/ios/HelloCordova.xcarchive/Products/Applications/HelloCordova.app/Frameworks/Sonect.framework/Headers/SNCConfiguration.h

```diff
@@ -0,0 +1,65 @@
+//
+//  SNCConfiguration.h
+//  SonectSDK
+//
+//  Created by Bruno Sousa on 21/12/2018.
+//  Copyright © 2018 Sonect. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SNCConfiguration : NSObject
+
+/**
+ A 2 digit ISO country code i.e. "ch" for Switzerland
+ */
+@property (nonatomic, nullable, copy, readonly) NSString* alpha2CountryCode;
+
+/**
+ The user's preferred currency i.e. "CHF" for Swiss Francs
+ */
+@property (nonatomic, nullable, copy, readonly) NSString* currency;
```

Comment:
Maybe better to use enum in configuration to let us control available currencies